### ISSUE TITLE
Invalid path alias

### ISSUE THREAD

**Issue by davidhellmann** [putyourlightson/craft-sprig#170]
### Describe the bug

tried to use the craft-cli but the following error occurred:

```php
Exception 'yii\base\InvalidArgumentException' with message 'Invalid path alias: @putyourlightson/sprig/controllers'

in /SITE_XXX/vendor/yiisoft/yii2/BaseYii.php:154

Stack trace:
#0 /SITE_XXX/vendor/yiisoft/yii2/base/Module.php(257): yii\BaseYii::getAlias('@putyourlightso...')
#1 /SITE_XXX/vendor/yiisoft/yii2/console/controllers/HelpController.php(245): yii\base\Module->getControllerPath()
#2 /SITE_XXX/vendor/yiisoft/yii2/console/controllers/HelpController.php(240): yii\console\controllers\HelpController->getModuleCommands(Object(putyourlightson\sprig\Sprig))
#3 /SITE_XXX/vendor/yiisoft/yii2/console/controllers/HelpController.php(172): yii\console\controllers\HelpController->getModuleCommands(Object(craft\console\Application))
#4 /SITE_XXX/vendor/yiisoft/yii2/console/controllers/HelpController.php(192): yii\console\controllers\HelpController->getCommands()
#5 /SITE_XXX/vendor/yiisoft/yii2/console/controllers/HelpController.php(292): yii\console\controllers\HelpController->getCommandDescriptions()
#6 /SITE_XXX/vendor/yiisoft/yii2/console/controllers/HelpController.php(67): yii\console\controllers\HelpController->getDefaultHelp()
#7 [internal function]: yii\console\controllers\HelpController->actionIndex(NULL)
#8 /SITE_XXX/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#9 /SITE_XXX/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#10 /SITE_XXX/vendor/yiisoft/yii2/console/Controller.php(184): yii\base\Controller->runAction('', Array)
#11 /SITE_XXX/vendor/yiisoft/yii2/base/Module.php(534): yii\console\Controller->runAction('', Array)
#12 /SITE_XXX/vendor/yiisoft/yii2/console/Application.php(181): yii\base\Module->runAction('', Array)
#13 /SITE_XXX/vendor/craftcms/cms/src/console/Application.php(89): yii\console\Application->runAction('', Array)
#14 /SITE_XXX/vendor/yiisoft/yii2/console/Application.php(148): craft\console\Application->runAction('', Array)
#15 /SITE_XXX/vendor/yiisoft/yii2/base/Application.php(392): yii\console\Application->handleRequest(Object(craft\console\Request))
#16 /SITE_XXX/craft(22): yii\base\Application->run()
#17 {main}
```

### Versions
- Plugin version: 1.9.0
- Craft version: 3.7.14

**Comment by davidhellmann** [comment]
- Reinstall doesn't solve the problem. 
- Delete vendor folder and composer.lock and composer install doesn't solve the problem

**Comment by bencroker** [comment]
Thanks, fixed in https://github.com/putyourlightson/craft-sprig-core/commit/42454a508beccd7c4efd74822b04808b5eceaec7 and released in version 1.9.1.
